islandora_videojs.install

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Install hooks.
+ */
+
+/**
+ * Information about the islandora_videojs update.
+ *
+ * Also removes the old variable.
+ */
+function islandora_videojs_update_7000(&$sandbox) {
```

Comment:
Probably not really an issue, and I believe we've done this similarly elsewhere, but [the `x0xx` range is supposed to be used for updates from earlier _Drupal_ versions](https://api.drupal.org/api/drupal/modules!system!system.api.php/function/hook_update_N/7.x) (the bolded bit):

>  1 digit for Drupal core compatibility.
    1 digit for your module's major release version (e.g., is this the 7.x-1.* (1) or 7.x-2.* (2) series of your module?). **This digit should be 0 for initial porting of your module to a new Drupal core API.**
    2 digits for sequential counting, starting with 00.